src/os/view/market/appViewer/screenshot.tsx

```diff
@@ -8,14 +8,30 @@ import { RootState } from 'os/store'
 import { MultiStaticLoader } from 'os/components/staticLoader'
 
 import imgError from 'os/static/images/error-image.svg'
+import { defaultImage } from 'app/static.app'
```

Comment:
Definitely the incorrect way to load an application's images. Please review the `staticLoader` for a panel loader.